### ISSUE TITLE
"look around at how lucky we are to be alive right now"

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/cgel#108]
An exclamative clause within a PP. Is "to be alive right now" a complement of "lucky"? How does that work?

**Comment by BrettRey** [comment]
Nice find!

There are subordinate exclamatives, though I'm not sure I had really internalized that fact.

1. _I’ll tell them what a good player she is._ (p. 62)
2. _I was amazed what a fuss he made._ (p. 545) [could equally be _amazed at_ or _amazed about_]
3. I realise what a bargain it is. (p. 854)
etc.

Other prepositions:
5. _It has reminded me of how important it is to maintain control of copyrights._
6. _It makes me reflect on how important it is to be careful._
7. _you'll be impressed by how easy it is to use_
8. _critics have pointed to how difficult it is to establish the adaptive function_
9. _Nobody can prepare you for how difficult it is to move out_
10. _I've been frustrated with how difficult it is to do things_

In terms of the structure, the canonical counterpart is _We are _(_very_)_ lucky to be alive right now_ and the exclamative phrase is fronted, so yes, the infinitival is licensed by _lucky_.

**Comment by nschneid** [comment]
What I'm not sure about is the whether only a part of the AdjP has been targeted for fronting:

[Clause Prenucleus:[how lucky]<sub>i</sub> we are [AdjP Head:__<sub>i</sub> Comp:[to be alive right now]]]

Or whether the whole thing is fronted and then there is postposing of the heavy complement phrase:

[Clause 
&nbsp;&nbsp;&nbsp;  Head:[Clause Prenucleus:[AdjP Head:[how lucky] Comp:\_\_<sub>j</sub>]<sub>i</sub> we are PredComp:\_\_<sub>i</sub>] 
&nbsp;&nbsp;&nbsp;  Postnucleus:[to be alive right now]<sub>j</sub>
]

**Comment by BrettRey** [comment]
So, not just a problem within the PP. Would you say it's similar to:

1. _What time should I arrive that would be better?_
2. _Which one did he choose of the three you offered?_
3. _Consider the knowledge he had about the past._

**Comment by nschneid** [comment]
Yeah I suppose so. Things that would be nonprojective in a dependency parse.

**Comment by nschneid** [comment]
<img width="451" alt="image" src="https://github.com/nert-nlp/cgel/assets/985263/65015fa6-defd-4d1b-bf92-d56ce74c434b">

**Comment by nschneid** [comment]
^ This suggests the postnucleus is extracted from inside the WH-phrase prenucleus, which is weird but (it seems to me) less weird than the reverse.

**Comment by BrettRey** [comment]
The reverse being extracting the prenucleus from the post nucleus? That hadn't occurred to me and strikes me as very weird. The other option I was wondering about, but not seriously, is somehow leaving the complement in the _are_ VP and extracting only the head. But your solution seems much better than that.

**Comment by BrettRey** [comment]
The reverse being extracting the prenucleus from the post nucleus? That hadn't occurred to me and strikes me as very weird. The other option I was wondering about, but not seriously, is somehow leaving the complement in the _are_ VP and extracting only the head. But your solution seems much better than that.

**Comment by BrettRey** [comment]
Can you paste the LaTeX for this?

**Comment by nschneid** [comment]
Pushed to the `datasets/oneoff` directory:
https://github.com/nert-nlp/cgel/blob/main/datasets/oneoff/lookaround.cgel
https://github.com/nert-nlp/cgel/blob/main/datasets/oneoff/tex/lookaround.tex
https://github.com/nert-nlp/cgel/blob/main/datasets/oneoff/pdf/lookaround.pdf

**Comment by BrettRey** [comment]
Thanks!
